README.md

```diff
@@ -20,11 +27,13 @@ Use this issue type if you are looking for someone to help you maintain or updat
 
 ## Progress and roadmap
 
+We will organize the issues into [projects](https://github.com/umbraco/Umbraco.Packages/projects) and work on one project at a time.
+
 We also use this repository to track our progress and to display a [roadmap](https://github.com/umbraco/Umbraco.Packages/projects/3) for the Package team. The roadmap is found under the projects tab, and it will contain other projects in a Now - Next - Later structure.
 
 ## Get in touch!
 
-If you want to get in touch with the team about something not covered by our issues then feel free to reach out to any of us on Twitter, or write an email to packages@umbraco.com.
+If you want to get in touch with the team about something not covered by our issues then feel free to reach out to the team on Twitter [@umbracoPackages](https://twitter.com/umbracoPackages) (DMs are open), or write an email to packages@umbraco.com.
```

Comment:
Thanks @NikRimington, good idea to add a link to that page. The twitter profiles of each team member are on that Our page so I think it might be easier to keep this page simple and just have the one here. One less place to edit when the team line up changes!  About to submit an update to this with a link to the Our page